requirements/pkgutils.txt

```diff
@@ -1,4 +1,4 @@
-setuptools>=20.6.7
+setuptools==58.2.0
```

Comment:
Pinning `setuptools<59.8.0` worked for me.

Reply:
i think i have to restrict the version in github action as well

Reply:
see https://github.com/celery/kombu/blob/master/.github/workflows/ci.yaml#L21

Reply:
https://github.com/celery/kombu/pull/1472